list-of-adopters.md

```diff
@@ -24,6 +24,7 @@ following projects/spaces have adopted this Code of Conduct -
 * [ocsigen/ocsigen-toolkit](https://github.com/ocsigen/ocsigen-toolkit) in [ocsigen/ocsigen-toolkit#224](https://github.com/ocsigen/ocsigen-toolkit/pull/224)
 * [ocsigen/ocsigenserver](https://github.com/ocsigen/ocsigenserver) in [ocsigen/ocsigenserver#227](https://github.com/ocsigen/ocsigenserver/pull/227)
 * [ocsigen/tyxml](https://github.com/ocsigen/tyxml) in [ocsigen/tyxml#318](https://github.com/ocsigen/tyxml/pull/318)
+* [OCurrent organisation](https://github.com/ocurrent) see [the organisation's profile](https://github.com/ocurrent/.github)
```

Comment:
Thanks! Do we still want to keep `current-bench` as a separate entry?

Reply:
Sounds good to me.

Reply:
Cool, it's also nice to know that GitHub gets the precedence of organization CoC vs repo CoC right: The "Code of conduct" link in the About section of the `current-bench` repo links to the repo CoC; the one in the other repos links to the organization CoC (it's something I've been wondering and now I know).